### PR TITLE
Bug 1402551, added steps about trusting the certificate

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1402551

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape PTAL

**Comment by ahardin-rh** [comment]
@openshift/team-documentation PTAL

**Comment by mburke5678** [comment]
Do we know what versions of Docker this applies to? I see the Docker page linked in the defect doesn't list specific versions, so.... 
LGTM

File: install_config/registry/securing_and_exposing_registry.adoc [hunk 1]
```diff
@@ -193,6 +193,27 @@ cf2616975b4a: Image successfully pushed
 Digest: sha256:3662dd821983bc4326bee12caec61367e7fb6f6a3ee547cbaff98f77403cab55
 ----
 
+. When using authentication, some versions of Docker also require you to trust the
+certificate at the OS level.
```
Comment (bfallonf) [inline install_config/registry/securing_and_exposing_registry.adoc]: Is "require you to trust" the correct working? Would it be more accurate to say "...require you to configure your cluster to trust..."?

File: install_config/registry/securing_and_exposing_registry.adoc [hunk 2]
```diff
@@ -193,6 +193,27 @@ cf2616975b4a: Image successfully pushed
...
+$ cp /etc/origin/master/ca.crt /etc/pki/ca-trust/source/anchors/myregistrydomain.com.crt
+----
+
+.. Run:
++
+----
+$ update-ca-trust enable
+----
+
+.. Restart the Restart the Docker service for the changes to take effect:
```
Comment (bfallonf) [inline install_config/registry/securing_and_exposing_registry.adoc]: Double "Restart the..."

**Comment by bfallonf** [comment]
@ahardin-rh a couple nits, but otherwise LGTM :truck:

**Comment by ahardin-rh** [comment]
Updated and also applied some edits from @adellape as discussed over IRC

**Comment by mburke5678** [comment]
LGTM

**Comment by ahardin-rh** [comment]
QE approved in the BZ

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../nstall_config/registry/securing_and_exposing_registry.adoc#install-config-registry-securing-exposing[Setting up the Registry -> Securing and Exposing the Registry]
|Added steps to the  xref:../install_config/registry/securing_and_exposing_registry.adoc#securing-the-registry[Securing the Registry] section on configuring your cluster to trust the certificate at the OS level.
%

**Pull request merged**